### ISSUE TITLE
Navigation landmarks must have ARIA labels and div elements need ARIA roles

### ISSUE THREAD

**Issue by Mankro** [apluslms/a-plus#751]
Check that A+ navigation landmarks have aria labels (or aria-labelledby) and that div based navigation has the correct aria role.

**Comment by skulonen** [comment]
@Mankro  I'm not super familiar with ARIA roles and labels, but I checked and found these two issues.

1) This part is intended for navigation, but is neither a nav element nor has a navigation role.
![image](https://user-images.githubusercontent.com/28843787/114033779-39576300-9886-11eb-8ffa-7c013cabf32b.png)

2) These blue blocks are nav elements and they have no ARIA labels. I don't think that they're intended to be navigation elements, though. Perhaps we should change their element type?
![image](https://user-images.githubusercontent.com/28843787/114034318-c0a4d680-9886-11eb-9ccd-aafe562405fe.png)

**Comment by Mankro** [comment]
@skulonen ARIA labels are the "user visible" names of the navigation elements for accessible technology (AT). If the label text is also a normal visible element in the page, then aria-labelledby is used. Otherwise, the label for AT only can be defined with aria-label. The label is visible to the user (i.e., screen readers read them aloud) and must be translated.

For example, the left-side course menu should be a navigation landmark and it must have an ARIA label.

1. The course page breadcrumb should have the ARIA role navigation, so you shall add that.
2. This is tricky. Since the "my submissions" dropdown is used to *navigate* to the feedback of each submission, I say this exercise points bar **is** navigation. Nav elements are automatically ARIA navigation landmarks, but the label must be added.

**Comment by Mankro** [comment]
You can take a look at old accessibility pull requests and see what was done.
https://github.com/apluslms/a-plus/pull/693 
https://github.com/apluslms/a-plus/pull/688
